hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/authorization/ValidateAccessToken.java

```diff
@@ -88,4 +99,5 @@ public void process(Exchange exchange)
         // Print out the token claims set
         logger.info("TOKEN PAYLOAD: " + claimsSet.toJSONObject());
```

Comment:
I realize this isn't part of the commit but we should use standard SLF4J String substitution (which you've done with all the recent additions).